packages/terra-functional-testing/tests/jest/commands/axe/inject.test.js

```diff
@@ -15,14 +15,14 @@ describe('Inject Axe', () => {
 
   it('should inject axe into the document with the provided options', () => {
     const mockExecute = jest.fn();
-    const mockRules = { rules: [{ id: 'mock-rule', enabled: true }] };
+    const mockRules = { rules: [{ enabled: true, id: 'mock-rule' }] };
```

Comment:
Curious what the reason is for swapping these? It's more natural to me to see the rule name first followed by their attributes.

Reply:
This is the order the objects will be defined for the real axe inject due to how the map generates the rules array. The id is always appended. For this test it doesn't matter, I've reverted the changes to how it was before